keepalived/core/snmp.c

```diff
@@ -151,7 +151,11 @@ snmp_scalar(struct variable *vp, oid *name, size_t *length,
 		long_ret = global_data->linkbeat_use_polling?2:1;
 		return (u_char *)&long_ret;
 	case SNMP_LVSFLUSH:
+#ifdef _WITH_LVS_
```

Comment:
The general approach with keepalived appears to be that if a feature isn't enabled, then no response is given for OIDs that aren't supported. That means the block should look more like:
```
#ifdef _WITH_LVS_
      case SNMP_LVSFLUSH:
      .
      .
      return ...
#endif
```
My updated patch incorporates this change.